sos/plugins/dnf.py

```diff
@@ -60,4 +66,17 @@ def setup(self):
         if self.get_option("history"):
             self.add_cmd_output("dnf history")
 
+        if self.get_option("history-info"):
+            history = self.call_ext_prog("dnf history")
```

Comment:
> But.. I don't really like the proliferation of options here - experience shows that users don't tend to find them unless pointed to them explicitly (which somewhat removes the point of having them - if a 2nd touch is needed to re-request sos data with -k dnf.history* then we almost might just as well ask for dnf history-whatever itself..).

Sorry I didn't reply to this point earlier. My current incentive is ovirt-log-collector, which calls sosreport, and will then be able to pass these options as needed, when available. For general/default use of sosreport, not sure. I think there is value in adding them always by default, as they do not take an unreasonable amount of time and quite little space. But I am not sure I am in a position to decide - it's the maintainer who knows what kinds of requests/complaints current users have, and might be able to better expect whether adding this amount of time will increase the "sosreport is too slow" complaints more than the win in less "I already sent a sosreport, do I really need to run it again? Can't you make it get you all you need at once?" kind of complaints, or the other way around...